prompt.go

```diff
@@ -11,20 +11,41 @@ import (
 func (i *InitActionCmd) setParams(dirs []string) error {
 	var err error
 	if i.DirPath == "" {
+		prompt := promptui.Select{
+			Label: "Choose driver",
+			Items: []string{"mysql", "postgres", "mariadb", "sqlite"},
+			Stdin: i.stdin,
+		}
+		if _, i.Driver, err = prompt.Run(); err != nil {
+			return err
+		}
+	}
+	switch {
+	case i.DirPath == "" && len(dirs) == 0:
+		prompt := promptui.Prompt{
+			Label: "Enter the path of the migration directory in your repository",
+			Stdin: i.stdin,
+		}
+		if i.DirPath, err = prompt.Run(); err != nil {
+			return err
+		}
+	case i.DirPath == "" && len(dirs) > 0:
+		opts := append(dirs, "provide another path")
 		prompt := promptui.Select{
 			Label: "Choose migration directory",
-			Items: dirs,
+			Items: opts,
 			Stdin: i.stdin,
 		}
 		if _, i.DirPath, err = prompt.Run(); err != nil {
 			return err
 		}
-		prompt = promptui.Select{
-			Label: "Choose driver",
-			Items: []string{"mysql", "postgres", "mariadb", "sqlite"},
+	}
```

Comment:
You can move that switch into the upper if since all cases require i.DirPath to be zero.